Yoast/YoastSEO.js
# Account for isApplicable in fullTextTests

## Summary

This PR can be summarized in the following changelog entry:

* Improves full text tests by taking `isApplicable` of the assessments into account

## Relevant technical choices:

* Removes the `assessmentsToRun` functionality (introduced in https://github.com/Yoast/YoastSEO.js/pull/1976), which served the same purpose as `isApplicable`, but was local to the full text tests.

## Test instructions

This PR can be tested by following these steps:

* Run the test sequence with `yarn test` and observe that nothing fails.

Fixes #2010
